django-verdant/basicauth/compat.py

```diff
@@ -0,0 +1,17 @@
+import sys
+
+import django
+
+PY2 = sys.version_info[0] == 2
+PY3 = sys.version_info[0] == 3
+
+DJ18 = django.VERSION[:2] == (1, 8)
+
+if PY2:
+    from urllib import unquote_plus  # NOQA
+else:
+    from urllib.parse import unquote_plus  # NOQA
+
+
+class MiddlewareMixin(object):
+    pass
```

Comment:
I've had to patch this for 1.9, not able to import `from django.utils.deprecation import MiddlewareMixin`
https://github.com/hirokiky/django-basicauth/blob/master/basicauth/compat.py#L20